### ISSUE TITLE
pytorch.onnx avg_pool2d (ceil_mode not supported)

### ISSUE THREAD

**Issue by fwz-fpga** [pytorch/pytorch#14399]
## ❓ Questions and Help
When will torch.onnx support avg_pool2d (ceil_mode=True) .

By the way, another problem is onnx support softmax (only dim=-1 supported), what's this "dim=-1" mean.
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)
